[2025-07-05T11:44:56.829+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2025-07-05T11:44:56.841+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: practice_etl_pipeline.extract manual__2025-07-05T11:44:52.485505+00:00 [queued]>
[2025-07-05T11:44:56.845+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: practice_etl_pipeline.extract manual__2025-07-05T11:44:52.485505+00:00 [queued]>
[2025-07-05T11:44:56.846+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 1
[2025-07-05T11:44:56.854+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): extract> on 2025-07-05 11:44:52.485505+00:00
[2025-07-05T11:44:56.857+0000] {clientserver.py:570} INFO - Closing down clientserver connection
[2025-07-05T11:44:56.858+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61: DeprecationWarning: This process (pid=2535) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-07-05T11:44:56.859+0000] {standard_task_runner.py:63} INFO - Started process 2709 to run task
[2025-07-05T11:44:56.860+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'practice_etl_pipeline', 'extract', 'manual__2025-07-05T11:44:52.485505+00:00', '--job-id', '19', '--raw', '--subdir', 'DAGS_FOLDER/main.py', '--cfg-path', '/tmp/tmpy2obl5x8']
[2025-07-05T11:44:56.862+0000] {standard_task_runner.py:91} INFO - Job 19: Subtask extract
[2025-07-05T11:44:56.896+0000] {task_command.py:426} INFO - Running <TaskInstance: practice_etl_pipeline.extract manual__2025-07-05T11:44:52.485505+00:00 [running]> on host 6802b12b52ea
[2025-07-05T11:44:56.944+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='practice_etl_pipeline' AIRFLOW_CTX_TASK_ID='extract' AIRFLOW_CTX_EXECUTION_DATE='2025-07-05T11:44:52.485505+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-07-05T11:44:52.485505+00:00'
[2025-07-05T11:44:56.945+0000] {taskinstance.py:430} INFO - ::endgroup::
[2025-07-05T11:44:56.953+0000] {logging_mixin.py:188} INFO - [] файлы в папке
[2025-07-05T11:44:57.014+0000] {python.py:237} INFO - Done. Returned value was: None
[2025-07-05T11:44:57.015+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2025-07-05T11:44:57.020+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=practice_etl_pipeline, task_id=extract, run_id=manual__2025-07-05T11:44:52.485505+00:00, execution_date=20250705T114452, start_date=20250705T114456, end_date=20250705T114457
[2025-07-05T11:44:57.032+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2025-07-05T11:44:57.047+0000] {taskinstance.py:3503} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-07-05T11:44:57.049+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
